### PR TITLE
Release/v0.47.2

### PR DESCRIPTION
**Pull request by jeremyletang**
### **_Just checking the CI, please do not merge_**

close #4499

### CODE REVIEW

**Review by pscott31 — APPROVED** [review]
It looks good to me, but I'm not sure exactly how to check what should be included in the release, so perhaps wait for approval from someone who knows what's going on!

**Review by ze97286 — APPROVED** [review]

**Pull request closed without merging**
